chart/env/prod.yaml

```diff
@@ -213,9 +213,9 @@ workers:
         memory: "8Gi"
     tolerations: []
   -
-    deployName: "split-names"
+    deployName: "split-names-from-streaming"
     maxJobsPerNamespace: 20
-    workerOnlyJobTypes: "/split-names"
+    workerOnlyJobTypes: "/split-names-from-streaming"
```

Comment:
wow, we missed this. The worker was not doing anything since the job did not exist